util-core/src/test/scala/com/twitter/util/CancellableTest.scala

```diff
@@ -3,6 +3,39 @@ package com.twitter.util
 import org.scalatest.WordSpec
 
 class CancellableTest extends WordSpec {
```

Comment:
Hi @muachilin, thanks for these changes! Would you mind changing this over to FunSpec? This is noted in our [CONTRIBUTING.md](https://github.com/twitter/util/blob/develop/CONTRIBUTING.md#testing). I apologize that this file wasn't moved over to FunSpec, but we would greatly appreciate if would make that change.

Reply:
Okay!! :)